extensions/wikia/ArticleVideo/scripts/featured-video.jwplayer.moat-tracking.js

```diff
@@ -2,17 +2,34 @@ define('wikia.articleVideo.featuredVideo.moatTracking', [
 	'ext.wikia.adEngine.adContext',
 	'wikia.window'
 ], function (adContext, win) {
-	return function(player) {
```

Comment:
It is going to break mobile-wiki.

https://github.com/Wikia/mobile-wiki/blob/6972127e4943353382db38da045efded7d667c6a/app/modules/video-players/jwplayer.js#L25

Reply:
Fixed in next commit https://github.com/Wikia/app/pull/14960/commits/9f71ffbe784748b5653ba8e778cec791df741f0d#diff-e2c7dd9a51618eb669f4323d534c13b9R36